src/modlunky2/ui/levels.py

```diff
@@ -1014,8 +1004,150 @@ def canvas_click_secondary(event, canvas):
         )
         self.tree_files.bind("<ButtonRelease-1>", self.tree_filesitemclick)
 
+    def reset(self):
+        print("Resetting..")
+        for i in self.tree_levels.get_children():
+            self.tree_levels.delete(i)
+        try:
+            self.canvas.delete("all")
+            self.canvas_dual.delete("all")
+            self.canvas.grid_remove()
+            self.canvas_dual.grid_remove()
+            self.foreground_label.grid_remove()
+            self.background_label.grid_remove()
+        except:
+            print("canvas does not exist yet")
+
+
+    def load_packs(self):
+        self.reset()
+        print("loading packs")
+
+        for i in self.tree_files.get_children():
+            self.tree_files.delete(i)
+        self.tree_files.heading("#0", text="Select Pack")
+        self.icons_packs = []
+        i = 0
+        for filepath in glob.iglob(str(self.overrides_path) + '/*/'):
+            self.icons_packs.append(ImageTk.PhotoImage(Image.open("modlunky2/static/images/folder.png").resize((25,25))))
```

Comment:
These need to be `BASE_DIR / "static/images/folder.png"` or they'll only work if you run modlunky relative to those files

Reply:
Uploading changes now, altho I'm unable to replicate your error